src/app/container/view/view.component.css

```diff
@@ -12,9 +12,4 @@
  *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  *    See the License for the specific language governing permissions and
  *    limitations under the License.
- */
-
-td {
-  word-wrap:break-word;
-  word-break:break-all;
-}
+ */
```

Comment:
Is there any point to this empty file anymore?

Reply:
There is little point to this empty file except for the corresponding  component currently pointing to this.  We have alot of empty css files due to angular cli component generation.  I could remove this file, but it would only make sense if we removed the other empty css files as well for consistency.

In the event that we need to add component-specific styles, we need to re-create the file with the same name.

I personally would rather leave it as it is, takes little/no space with little performance hit.